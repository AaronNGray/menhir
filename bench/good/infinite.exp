File "infinite.mly", line 10, characters 0-8:
Warning: infinite generates the empty language.
Warning: one state has an end-of-stream conflict.
File "infinite.mly", line 7, characters 2-3:
Warning: production dummy -> A is never reduced.
Warning: in total, 1 productions are never reduced.
