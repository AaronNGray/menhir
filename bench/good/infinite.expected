%start dummy
%token A

%type <unit> dummy
%%

dummy:
| A
    { () }
| A infinite
    { () }

infinite:
| A infinite
    { () }

%%



